lib/rules/no-attrs-in-components.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const utils = require('../utils/utils');
+
+const MESSAGE = 'Do not use this.attrs';
+
+//------------------------------------------------------------------------------
+// General rule - Don't use this.attrs
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Disallow usage of this.attrs in components',
+      category: 'General',
+      recommended: true
+    },
+    fixable: null,
+  },
+
+  create(context) {
+    return {
+      MemberExpression(node) {
```

Comment:
Although it's only being ported I'm thinking whether we should scope this check to component files only and do not throw error if it's used in util or anywhere else. What do you think? Anyway I'm ok with postponing this update at the moment.